src/components/Tab/Tab.tsx

```diff
@@ -14,20 +14,25 @@ export interface Props {
   activeTabId?: string;
   // Callback function to be called when tab is clicked/selected
   onClick?(): void;
+  onFocus?(): void;
```

Comment:
@Farhan-emgage what's the purpose of this prop?

Reply:
@weegrom this is for tabs. actually this tabs are in <div></div>. and the content is opening on click event. but for accessibility by keyboard it should be open on focus also. because of this reason i added onFocus() prop

Reply:
@Farhan-emgage can you please investigate if there is a way to use a single event for both mouse and keyboard. It seems odd to required both onClick and onFocus. Also, wouldn't OnFocus simply select the tab, and then hitting "Enter" open the tab?

Reply:
@weegrom Yes, this is possible. but for that we have to place this tab in either in anchor tag or it should be inside of button.

Reply:
@Farhan-emgage what was the final decision here. I'd like to get this PR closed.